docs/src/pages/components/slider/CustomizedSlider.tsx

```diff
@@ -76,80 +76,81 @@ const IOSSlider = withStyles({
       },
     },
   },
-  active: {},
-  valueLabel: {
+
+  '& .MuiSlider-valueLabel': {
     left: 'calc(-50% + 12px)',
     top: -22,
     '& *': {
       background: 'transparent',
       color: '#000',
     },
   },
-  track: {
+
```

Comment:
Are the spaces added by the TypeScript -> JavaScript generation script?

If not, what coding style convention should we adopt? A blank line between selector seems common in SASS land.

Reply:
I think I did the changes, but for the classes selectors that we have now, I would propose blank line before each new selector as a convention. It's much more readable that way